### PR TITLE
Add C4 components to the documentation page

### PR DESCRIPTION
**Pull request by tranchitella**

### CODE REVIEW

**Comment by tranchitella** [comment]
@mwolff44 @fredericlepied @alexsosic 
Can you please review this PR to publish C4 docs on wazo-platform.org? The preview link is here:
https://zuul.wazo.community/logs/06/106/4c4fcd4d91cde4bdbd1958203a7753eed9cf7cc0/check/yarn-build-copy/5cab8cf/

**Comment by fredericlepied** [comment]
> @mwolff44 @fredericlepied @alexsosic
> Can you please review this PR to publish C4 docs on wazo-platform.org? The preview link is here:
> https://zuul.wazo.community/logs/06/106/4c4fcd4d91cde4bdbd1958203a7753eed9cf7cc0/check/yarn-build-copy/5cab8cf/

The direct URL is this one: ttps://zuul.wazo.community/logs/06/106/4c4fcd4d91cde4bdbd1958203a7753eed9cf7cc0/check/yarn-build-copy/5cab8cf/public/

**Comment by tranchitella** [comment]
> Looks great! I would prefer to have:
> 
> * "C4 Router API" in the "Programmable Platform" section.
> * "C4 SBC", "C4 Router", "C4 RTPEngine" in the "Internal platform" section.
> 
> Also we need the diagrams to describe the relations between the components. I can work with you on these.

Moved the components as you suggested.
Regarding the diagrams, I'll try to come up with something based on the other diagrams we have in the repo.

**Comment by tranchitella** [comment]
recheck

**Comment by tranchitella** [comment]
recheck

**Comment by tranchitella** [comment]
recheck

**Comment by tranchitella** [comment]
regate

File: content/c4-router/description.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+* Source IP and port
+* SIP Username/Password
+
+### Routing
+
+Class 4 equipments are located at the periphery of the network; it is necessary to implement routing mechanisms sometimes simple, sometimes sophisticated. The mechanism will depend on the desired services and may be different depending on the targets. Wazo C4 Router supports both inbound and outbound routing based on routing rules:
+
+* Authentication details
+* Source IP and port
+* Destination DID
```
Comment (mwolff44) [inline content/c4-router/description.md]: I will ad Inbound routing :

Comment (mwolff44) [reply content/c4-router/description.md]: And add a section with outbound routing

File: content/c4-router/description.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+The SIP router performs authentication, both for Carrier Trunks and for IPBX, to ensure only authorized calls are successfully routed to the destination.
+We support the following authentication methods:
+
+* Source IP and port
+* SIP Username/Password
+
+### Routing
+
+Class 4 equipments are located at the periphery of the network; it is necessary to implement routing mechanisms sometimes simple, sometimes sophisticated. The mechanism will depend on the desired services and may be different depending on the targets. Wazo C4 Router supports both inbound and outbound routing based on routing rules:
```
Comment (mwolff44) [inline content/c4-router/description.md]: I will add support source and destination routing decisions

File: content/c4-router/description.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+### High availability
+
+The Router component is a crucial component for all the calls; it must be resilient to errors and outages. The Wazo Router provides Redis-based synchronization of the dialogs to offer High Availability and horizontal scaling.
+
+## Part of the Wazo Platform C4
+
+A Class 4 Softswitch routes large volumes of usually long-distance VoIP calls. For businesses that want to interconnect their VoIP servers, a Class 4 Softswitch is used to relay VoIP traffic and services over multiple IP networks. C4 soft switches provide intelligent call routing, which reduces congestion, latency, and costs while improving the quality of VoIP calls. They have several security features to protect the C5 switches.
+
+The main characteristics of a C4 Softswitch are:
```
Comment (mwolff44) [inline content/c4-router/description.md]: the main "security features" of ...

File: content/c4-router/description.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+* route large volume of calls
+* protocol support and conversion
+* transcoding
+* billing interface
+* security management
+* call authentication
```
Comment (mwolff44) [inline content/c4-router/description.md]: and authorization

File: content/c4-router/diagram-c4.puml [hunk 5]
```diff
@@ -0,0 +1,37 @@
+@startuml
+!include C4_Container.puml
```
Comment (mwolff44) [inline content/c4-router/diagram-c4.puml]: Why to not integrate the full url : !includeurl https://raw.githubusercontent.com/RicardoNiepel/C4-PlantUML/master/C4_Container.puml
 ?

**Pull request merged**